src/DynamoCoreWpf/UI/GuidedTour/GuidesValidationMethods.cs

```diff
@@ -328,6 +329,47 @@ internal static void SubscribeSearchForPackagesOption(Step stepInfo, StepUIAutom
                 foundUIElement.Click -= SearchForPackage_Click;
         }
 
+        /// <summary>
+        /// This method will subscribe the Next button from the Step8 Popup for clicking the Package already installed (then it will be expanded).
+        /// </summary>
+        /// <param name="stepInfo">Information about the Step</param>
+        /// <param name="uiAutomationData">Specific UI Automation step that is being executed</param>
+        /// <param name="enableFunction">it says if the functionality should be enabled or disabled</param>
+        /// <param name="currentFlow">The current flow of the Guide can be FORWARD or BACKWARD</param>
+        internal static void SubscribeNextButtonClickEvent(Step stepInfo, StepUIAutomation uiAutomationData, bool enableFunction, GuideFlow currentFlow)
+        {
+            CurrentExecutingStep = stepInfo;
+            var nextbuttonFound = Guide.FindChild((stepInfo.StepUIPopup as PopupWindow).mainPopupGrid, "NextButton") as Button;
```

Comment:
Maybe define `NextButton` separately?

Reply:
@QilongTang The hardcoded text "NextButton" was moved to the json file to a handler so we can create unit test for checking names easily and also for subscribing the Button.Click event is using the ManageEventHandler method.
See the commit: https://github.com/DynamoDS/Dynamo/pull/12312/commits/0397fe711f7a87f84e45c77e4fc9ddd615ff2c2b